site/installing.md

```diff
@@ -173,26 +173,18 @@ It is recommended that you run Scope natively in your Kubernetes cluster using t
 
 The simplest way to get the latest release of Scope deployed onto a Kubernetes cluster is the following:
 
-```
-kubectl create -f 'https://scope.weave.works/k8s-gen/weavescope.json' --validate=false
-```
+    kubectl create -f 'https://scope.weave.works/k8s-gen/weavescope.json' --validate=false
```

Comment:
Yes, I'm afraid it's currently needed. Due to a bug in Kubernetes, but we can figure it out later, if it really bothers people.